packages/common/src/search/wellKnownCatalog.ts

```diff
@@ -129,46 +147,122 @@ function getWellknownItemCatalog(
     "item",
     options?.collectionNames
   );
-  switch (name) {
+  switch (catalogName) {
     case "myContent":
-      checkUser(name, options);
+      validateUserExistence(catalogName, options);
       catalog = buildCatalog(
         i18nScope,
-        name,
+        catalogName,
         [{ owner: options.user.username }],
-        collections
+        collections,
+        "item"
       );
       break;
     case "favorites":
-      checkUser(name, options);
+      validateUserExistence(catalogName, options);
       catalog = buildCatalog(
         i18nScope,
-        name,
+        catalogName,
         [{ group: options.user.favGroupId }],
-        collections
+        collections,
+        "item"
       );
       break;
     case "organization":
-      checkUser(name, options);
+      validateUserExistence(catalogName, options);
       catalog = buildCatalog(
         i18nScope,
-        name,
+        catalogName,
         [{ orgid: options.user.orgId }],
-        collections
+        collections,
+        "item"
       );
       break;
     case "world":
       catalog = buildCatalog(
         i18nScope,
-        name,
+        catalogName,
         [{ type: { not: ["code attachment"] } }],
-        collections
+        collections,
+        "item"
       );
       break;
   }
   return catalog;
 }
 
+/**
+ * Get a group catalog based on the name and optional requests
+ * @param i18nScope Translation scope to be interpolated into the catalog
+ * @param catalogName Name of the catalog requested
+ * @param options An opitional IGetWellKnownCatalogOptions definition JSON object
+ * @returns A group IHubCatalog definition JSON object
+ */
+function getWellknownGroupCatalog(
+  i18nScope: string,
+  catalogName: WellKnownCatalog,
+  options?: IGetWellKnownCatalogOptions
+): IHubCatalog {
+  i18nScope = dotifyString(i18nScope);
+  let catalog;
+
+  switch (catalogName) {
+    case "editGroups":
+      validateUserExistence(catalogName, options);
+      catalog = buildCatalog(
+        i18nScope,
+        catalogName,
+        [{ capabilities: ["updateitemcontrol"] }],
+        [],
+        "group"
+      );
+      // because collections are needed in arcgis-hub-catalog and
+      // "searchGroups" allows 'q: "*"', we use this as the collection
+      catalog.collections = [
+        {
+          targetEntity: "group",
+          key: catalogName,
+          label: catalogName,
+          scope: {
+            targetEntity: "group",
+            filters: [
+              {
+                predicates: [{ q: "*" }],
+              },
+            ],
+          },
+        },
+      ];
+      break;
+    case "allGroups":
+      validateUserExistence(catalogName, options);
+      catalog = buildCatalog(
+        i18nScope,
+        catalogName,
+        [{ group: options.user.orgId }],
```

Comment:
This does not look right - a `group` predicate would expect to take a group id, not an `orgId`. In my testing directly against the API, I was using `capabilities: [""]` - not sure if that will work as a predicate though.

Reply:
Ah this was something I totally missed! Just tested `capabilities: [""]` and it seemed to work

![image](https://user-images.githubusercontent.com/16672774/234692969-8cdc4601-05b5-4b35-a896-8b1f1525e445.png)